modules/administration/pages/maintenance-window.adoc

```diff
@@ -86,3 +92,16 @@ zypper migration
 ----
 
 This procedure is also known as a Service Pack (SP) migration.
+
+
+
+== Inter-Server Synchronization Slave Server
+
+Inter-server synchronization slave servers should be updated as soon as
+the update of the main server (master server) is completed.
+
+
+
+== {productname} Proxy
+
+Proxies should be updated after all Server updates are complete.
```

Comment:
> Proxies should be updated after all {productname} Server updates are complete.

?

Reply:
I think this is correct, although it could be made clearer perhaps.

Basically we are saying that in a 3.2 to 4.0 migration you should upgrade the Server first, then any Proxy.